### PR TITLE
Bg 11 functionality size

### PR DESCRIPTION
**Pull request by romelu**

### CODE REVIEW

File: code/js/rcalc-math.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // utility
 const STRING_EMPTY = "";
-const NUMBER_NULL = "0";
-// buttons (operators and operands)
+const NUMBER_NULL = 0;
```
Comment (palomino7) [inline code/js/rcalc-math.js]: Sorry but 0 looks like ZERO not NULL.

File: code/js/rcalc-math.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ const ID_DISPLAY_INFO = "id-display-info";
 const COLOR_BLACK = "black";
```
Comment (palomino7) [inline code/js/rcalc-math.js]: We agreed on hexa format in past.

File: code/js/rcalc-personalize.js [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+const RCALC_DISPLAY_MAIN = "rcalc-display-main";
+const RCALC_DISPLAY_INFO = "rcalc-display-info";
 
 // changing the attribute href, which results in changing css file applied
 function swapCssFile(theme) {
   document.getElementById(ID_PAGE_STYLE).setAttribute("href", theme);
+}
+
+function changeSizeSmall() {
+  changeSize(RCALC_BUTTON, 20, 30, 30);
```
Comment (palomino7) [inline code/js/rcalc-personalize.js]: changeSize(RCALC_BUTTON, BUTTON_SIZE_SMALL_FONT_SIZE, BITTON_WIDTH_SMALL, BUTTON_HEIGHT_SMALL); 
do it similar for all

File: code/js/rcalc-personalize.js [hunk 4]
```diff
@@ -1,6 +1,43 @@
 const ID_PAGE_STYLE = "id-page-style";
+// classes
```
Comment (palomino7) [inline code/js/rcalc-personalize.js]: All three buttons(size, theme, localization) will have separate file with related logic.

File: code/js/rcalc-personalize.js [hunk 5]
```diff
@@ -1,6 +1,43 @@
...
+  changeSize(RCALC_BUTTON_OPERATOR, 70, 100, 100);
+  changeSize(RCALC_BUTTON_CLEAR, 70, 100, 100);
+  changeSize(RCALC_DISPLAY_MAIN, 70, 305, 100);
+  changeSize(RCALC_DISPLAY_INFO, 30, 410, 100);
+}
+
+function changeSize(rcalcCssClass, fontSize, width, height) {
+  var buttonsDisplays = document.getElementsByClassName(rcalcCssClass);
+  for (var i = 0; i < buttonsDisplays.length; i++) {
+    buttonsDisplays[i].style.fontSize = fontSize + "px";
```
Comment (palomino7) [inline code/js/rcalc-personalize.js]: Constant - SUFFIX_PX
Also you can create method addSuffixPx(size) where you will send fontSize etc.

File: code/js/rcalc-personalize.js [hunk 6]
```diff
@@ -1,6 +1,43 @@
...
+    buttonsDisplays[i].style.width = width + "px";
```
Comment (palomino7) [inline code/js/rcalc-personalize.js]: create one method where you send just one string as parameter and return result.

File: code/js/rcalc-math.js (hunk 1 above)
Comment (romelu) [reply code/js/rcalc-math.js]: renamed to NUMBER_ZERO

File: code/js/rcalc-math.js (hunk 2 above)
Comment (romelu) [reply code/js/rcalc-math.js]: hexa format only applied

File: code/js/rcalc-personalize.js (hunk 3 above)
Comment (romelu) [reply code/js/rcalc-personalize.js]: done for all

File: code/js/rcalc-personalize.js (hunk 4 above)
Comment (romelu) [reply code/js/rcalc-personalize.js]: separate files created rcalc-theme.js, rcalc-size.js, rcalc-language.js

File: code/js/rcalc-personalize.js (hunk 5 above)
Comment (romelu) [reply code/js/rcalc-personalize.js]: const SUFFIX_PX in the game

Comment (palomino7) [reply code/js/rcalc-personalize.js]: and new method not?

File: code/js/rcalc-language.js [hunk 7]
```diff
@@ -0,0 +1 @@
+﻿ // change language
```
Comment (palomino7) [inline code/js/rcalc-language.js]: Adding of this file should come later. :-)

Comment (romelu) [reply code/js/rcalc-language.js]: deleted ;-)

File: code/js/rcalc-personalize.js (hunk 5 above)
Comment (romelu) [reply code/js/rcalc-personalize.js]: of course new method also, here we have it

Comment (romelu) [reply code/js/rcalc-personalize.js]: function addSuffixPx(size) {
  var suffixPxAdded;
  suffixPxAdded = size + SUFFIX_PX;
  return (suffixPxAdded);
}

File: code/js/rcalc-size.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  var buttonsDisplays = document.getElementsByClassName(rcalcCssClass);
+  for (var i = 0; i < buttonsDisplays.length; i++) {
+    buttonsDisplays[i].style.fontSize = addSuffixPx(fontSize);
+    buttonsDisplays[i].style.width = addSuffixPx(width);
+    buttonsDisplays[i].style.height = addSuffixPx(height);
+  }
+}
+
+function addSuffixPx(size) {
+  var suffixPxAdded;
```
Comment (palomino7) [inline code/js/rcalc-size.js]: one line please ... too much rows for simple thing

Comment (romelu) [reply code/js/rcalc-size.js]: only think I can do here is delete one row with declaration of variable var suffixPxAdded, and declare the variable and assign a value to it in one row as this var suffixPxAdded = size + SUFFIX_PX;
Atom beautify does not let me to write it all in one row, after saving it it automatically formats it

File: code/js/rcalc-size.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  for (var i = 0; i < buttonsDisplays.length; i++) {
+    buttonsDisplays[i].style.fontSize = addSuffixPx(fontSize);
+    buttonsDisplays[i].style.width = addSuffixPx(width);
+    buttonsDisplays[i].style.height = addSuffixPx(height);
+  }
+}
+
+function addSuffixPx(size) {
+  var suffixPxAdded = size + SUFFIX_PX;
+  return (suffixPxAdded);
```
Comment (palomino7) [inline code/js/rcalc-size.js]: Again ... one line please and brackets are useless. 
You can return result directly.

Comment (romelu) [reply code/js/rcalc-size.js]: DONE, Thank you !!! one line only return size + SUFFIX_PX;

**Pull request merged**